### PR TITLE
Update to provide updated CSS instructions

### PR DESCRIPTION
**Pull request by benfrain**
Updated language server references based on #965
Also added basic rules override example.

Apologies if this is still not correct, unfamiliar with Lua!

### CODE REVIEW

File: lua/lspconfig/cssls.lua [hunk 1]
```diff
@@ -21,7 +21,77 @@ configs[server_name] = {
...
+Neovim does not currently include built-in snippets. vscode-css-language-server only provides completions when snippet support is enabled. To enable completion, install a snippet plugin and add the following override to your language client capabilities during setup.
+
+```lua
+--Enable (broadcasting) snippet capability for completion
+local capabilities = vim.lsp.protocol.make_client_capabilities()
+capabilities.textDocument.completion.completionItem.snippetSupport = true
+
+require'lspconfig'.cssls.setup {
+  capabilities = capabilities,
+}
```
Comment (mjlbach) [inline lua/lspconfig/cssls.lua:43]: Pretty much everything after this point is unecessary IMO

File: lua/lspconfig/cssls.lua [hunk 2]
```diff
@@ -21,7 +21,77 @@ configs[server_name] = {
   };
   docs = {
     description = [[
-https://github.com/vscode-langservers/vscode-css-languageserver-bin
+
+    
+https://www.npmjs.com/package/vscode-langservers-extracted
```
Comment (mjlbach) [inline lua/lspconfig/cssls.lua:26]: I would change this to the github URL of the project

**Comment by mjlbach** [comment]
Do you not also need to update `cmd` with the new binary name? I don't use this language server.

File: lua/lspconfig/cssls.lua (hunk 1 above)
Comment (benfrain) [reply lua/lspconfig/cssls.lua:43]: As a new user, I added the example as I found the original section  confusing/insufficient (with the nested defaults). I’m a fan of a cut and paste example but obviously go with what makes sense from your POV.

Comment (mjlbach) [reply lua/lspconfig/cssls.lua:43]: I think the cut/paste example is overly specific to your usecase.  Also one of the blocks basically duplicates what docgen does

Comment (benfrain) [reply lua/lspconfig/cssls.lua:43]: Hi, commit https://github.com/neovim/nvim-lspconfig/pull/971/commits/61ba064065b301657cc3ad31e26fd89795044d41 should have it as you desire.

File: lua/lspconfig/cssls.lua [hunk 3]
```diff
@@ -21,15 +21,17 @@ configs[server_name] = {
...
+
 ```sh
-npm install -g vscode-css-languageserver-bin
+npm i -g vscode-langservers-extracted
 ```
-
-Neovim does not currently include built-in snippets. `vscode-css-languageserver` only provides completions when snippet support is enabled.
-To enable completion, install a snippet plugin and add the following override to your language client capabilities during setup.
+    
+Neovim does not currently include built-in snippets. vscode-css-language-server only provides completions when snippet support is enabled. To enable completion, install a snippet plugin and add the following override to your language client capabilities during setup.
```
Comment (mjlbach) [inline lua/lspconfig/cssls.lua:34]: Out of curiosity, why did you remove the backticks around the server name?

File: lua/lspconfig/cssls.lua (hunk 1 above)
Comment (mjlbach) [reply lua/lspconfig/cssls.lua:43]: Lines 45 to 66 still need to be removed

File: lua/lspconfig/cssls.lua (hunk 3 above)
Comment (benfrain) [reply lua/lspconfig/cssls.lua:34]: Sheer stupidity!

File: lua/lspconfig/cssls.lua [hunk 4]
```diff
@@ -40,6 +42,7 @@ require'lspconfig'.cssls.setup {
   capabilities = capabilities,
 }
 ```
+
```
Comment (mjlbach) [inline lua/lspconfig/cssls.lua:45]: unnecessary space :)

**Comment by mjlbach** [comment]
@benfrain if you want, you can also do html and json language servers since these are also available from the same repo :) no pressure though. Thanks for the PR!

**Comment by benfrain** [comment]
Sure, should have a better idea what I'm doing now!
